setup.cfg

```diff
@@ -7,14 +7,15 @@ ignore = E501,E741,W503
 
 [flake8]
 max-line-length = 88
-ignore = E501,E741,W503
+ignore = E501,E741,W503,E203
 exclude = .git,__pycache__,docs/source/conf.py,old,build,dist
 
 [mypy]
 warn_redundant_casts = true
 disallow_untyped_calls = true
 disallow_untyped_defs = true
 no_implicit_optional = true
+ignore_missing_imports = true
```

Comment:
Why do we want to ignore this?

Reply:
The reason this is required is that there are no mypy headers for the `@backoff` decorator.

Reply:
Can we ignore this inline then globally?

Reply:
It is also possible to limit this per package instead for all imports. See example below in this file.